kolibri/plugins/learn/assets/src/constants.js

```diff
@@ -1,11 +1,10 @@
 // a name for every URL pattern
 const PageNames = {
-  TOPICS_ROOT: 'TOPICS_ROOT',
+  CHANNELS: 'CHANNELS',
```

Comment:
I'd prefer if the pageName mapped nicely with the URL.

Reply:
So, change all TOPICS references to CHANNELS? (including the URL?)

Reply:
As long as it's consistent. Topics makes more sense than channels, but `BROWSE` makes more sense than either... You browse content and topics...